.changelog/3428.breaking.md

```diff
@@ -0,0 +1 @@
+go/staking: Add support for beneficiary allowances
```

Comment:
Maybe a link to the ADR 0003 would be useful here.

Reply:
Added.